api/controllers/v1/slack.ts

```diff
@@ -10,6 +10,11 @@ function isUserEntitled(entitlementsObject: any): boolean {
   return (entitlementsObject?.repos?.length ?? 0) > 0;
 }
 
+function isRestrictedToDeploy(userId: string): boolean {
+  const prodDeploy = c.get<any>('prodDeploy');
+  return (prodDeploy.restrictedProdDeploy && !prodDeploy.entitledSlackUser.includes(userId));
```

Comment:
```suggestion
  const { restrictedProdDeploy, entitledSlackUsers } = c.get<any>('prodDeploy');
  return (restrictedProdDeploy && !entitledSlackUsers.includes(userId));
```

Sorry if it's annoying for me to nit pick on this twice - but realized it can be even cleaner 😅